src/badgehub-api-client/index.ts

```diff
@@ -6,9 +6,31 @@ interface BadgehubApp {
   slug: string;
   category_slug: string;
   user_name: string;
+  devices: BadgehubDevice[];
 }
 
-export async function getApps() {
+export async function getApps(category: string, device: string) {
+
+  let path = "/apps";
+
+  const searchParams = new URLSearchParams();
```

Comment:
If we use an optional options object as I suggested above, we can pass that in to the constructor of URLSearchParams and it will handle omitting values that are undefined for us.